groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group9/CobaltChain.groovy

```diff
@@ -2,19 +2,77 @@ import globals.Globals
 import static globals.CarbonGlobals.*
 
 ROASTER = recipemap('roaster')
+EBF = recipemap('electric_blast_furnace')
 PBF_RECIPES = recipemap("primitive_blast_furnace")
+FLUIDIZEDBR = recipemap('fluidized_bed_reactor')
+BR = recipemap('batch_reactor')
+ELECTROLYTIC_CELL = recipemap('electrolytic_cell')
+
+//ROAST RECOVERY OF As2O3
+//COBALTITE MODIFICATION
+mods.gregtech.electric_blast_furnace.removeByInput(120, [metaitem('dustCobaltite')], [fluid('oxygen') * 3000])
+
+
+EBF.recipeBuilder()
+        .inputs(ore('dustCobaltite'))
+        .fluidInputs(fluid('oxygen') * 9000)
+        .outputs(metaitem('dustImpureCobaltOxide'))
+        .chancedOutput(metaitem('dustArsenicTrioxide') * 5, 7500, 0)
+        .fluidOutputs(fluid('sulfur_dioxide') * 2000)
+        .EUt(Globals.voltAmps[1])
+        .blastFurnaceTemp(1023)
+        .duration(240)
+        .buildAndRegister()
+
+FLUIDIZEDBR.recipeBuilder()
+        .inputs(ore('dustCobaltite'))
+        .fluidInputs(fluid('oxygen') * 9000)
+        .outputs(metaitem('dustImpureCobaltOxide'))
+        .chancedOutput(metaitem('dustArsenicTrioxide') * 5, 9000, 0)
+        .fluidOutputs(fluid('sulfur_dioxide') * 2000)
+        .EUt(Globals.voltAmps[2])
+        .duration(120)
+        .buildAndRegister()
+
+BR.recipeBuilder()
```

Comment:
use cobalt chloride electrolysis tbh

Reply:
![image](https://github.com/user-attachments/assets/8475e777-984b-4dd3-b78f-2042b2256edc) 
(from UIlmann)
sulfates are less annoying to handle tho? you don't have to deal with volatile HCl and Cl